src/aks-preview/HISTORY.rst

```diff
@@ -13,6 +13,10 @@ Pending
 +++++++
 * Minimise the roles needed to introduce for Elastic SAN for enabling Azure Container Storage with elasticSan storagepool type.
 
+3.0.0b6
++++++++
+* Add option `WindowsAnnual` to `--os-sku` for `az aks nodepool add`.
```

Comment:
If you want to release a new version, please move the note from `Pending` (several lines above) here and update `VERSION` correspondingly in `setup.py`.

Reply:
Updated. Thanks for the comment.